planner/core/point_get_plan.go

```diff
@@ -389,7 +389,7 @@ func TryFastPlan(ctx sessionctx.Context, node ast.Node) (p Plan) {
 			p = fp
 			return
 		}
-		if fp := tryPointGetPlan(ctx, x); fp != nil {
+		if fp := tryPointGetPlan(ctx, x, false); fp != nil {
```

Comment:
Should be `true`？

Reply:
Changed to `x.LockTp != ast.SelectLockNone` now.